CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingPage.swift

```diff
@@ -7,10 +7,23 @@
 
 import Foundation
 
+/**
+ `OnboardingPageNew` represents the onboarding page with all it's properties.
```

Comment:
@vizaknaidavid  could you please remove the old `OnboardingPage` and rename `OnboardingPageNew` - > `OnboardingPage` changes in the documentation for this renaming can be done in this PR 😇